spec/components/candidate_interface/rejections_component_spec.rb

```diff
@@ -57,6 +58,33 @@
     end
   end
 
+  describe "when the rejection reason type is 'enic rejection_reasons'" do
```

Comment:
I think we need a bit more information here. Maybe something to explain that the qualification doesn't have an ENIC Reference